packages/nitro/types/shims.d.ts

```diff
@@ -9,3 +9,10 @@ declare module '#assets' {
   export function statAsset (id: string): Promise<AssetMeta>
   export function getKeys() : Promise<string[]>
 }
+
+declare module '#config' {
+  import type { PublicRuntimeConfig, PrivateRuntimeConfig, RuntimeConfig } from '@nuxt/kit'
+  export const privateConfig: PrivateRuntimeConfig
+  export const publicConfig: PublicRuntimeConfig
+  export default RuntimeConfig
```

Comment:
```suggestion
  export default const config: RuntimeConfig
```
Otherwise the interface is exported if I'm not mistaken.

Reply:
You're right that this needs to be tweaked:
```suggestion
  const runtimeConfig: RuntimeConfig
  export default runtimeConfig
```